index.html

```diff
@@ -25,6 +25,11 @@ <h1 class="name">Nest Meet</h1>
 
       <div class="header-spacer"></div>
     </div>
-    
+   
+
+    <footer class = "Digital-Nest-Footer">
```

Comment:
lets keep class names kebab-casing and leave it lower case
```suggestion
    <footer class="digital-nest-footer">
```